contacts24/contacts.py

```diff
@@ -42,12 +43,14 @@ def load_contacts_book() -> AddressBook:
 
 @input_error
 def add_contact(args: CommandArguments, contacts: AddressBook) -> str:
-    if args is None or len(args) < 2:
+    if args is None or len(args) < 4:
         raise AddContactInputError()
 
-    name, phone = args[:2]
+    name, phone, email, address = args[:4]
```

Comment:
@krisdzhus Do you want to add them all in 1 line?
This might be slighly confusing, I think, a lot of stuff to type and re-type if, say, email is broken. Maybe, we just add them all one by one, as with birthdays?

Reply:
@IhorNi optional could add complexity
Suggest to add lt as is
And add new task to optimize later today

Reply:
Agree, too much complexity, let's do it separatly, and assign email and address just like birthday in a decicated command for simplicity.

Reply:
And @krisdzhus as I missed in files below, so we are alinged on this)